components/status_dropdown/status_dropdown.tsx

```diff
@@ -105,13 +109,13 @@ export default class StatusDropdown extends React.PureComponent {
         );
     }
 
-    renderDropdownIcon = () => {
+    renderDropdownIcon = (): ReactNode => {
         return (
             <FormattedMessage
                 id='generic_icons.dropdown'
                 defaultMessage='Dropdown Icon'
             >
-                { (title) => (
+                { (title: any) => (
```

Comment:
Is it feasible to pick a narrower type here?

Reply:
hi @lieut-data , I already change data type to "string".